.nf-core.yml

```diff
@@ -8,3 +8,4 @@ bump-versions:
   rseqc/inferexperiment: False
   rseqc/innerdistance: False
   sortmerna: False
+repository_type: modules
```

Comment:
It's a bit strange that you would introduce this value here.

Reply:
I did not change anything in this file. Possibly has been done automatically?

Reply:
Probably from merging in master but then it shouldn't really show up as a diff 🤔

Reply:
Ah, prettier is complaining about this now. When you resolved conflicts you accidentally inserted this at the top and bottom of the file. Please remove.

```suggestion
```